packages/components/components/elvis-breadcrumb/src/react/elvia-breadcrumb.tsx

```diff
@@ -20,18 +20,10 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = function ({
   webcomponent,
   ...rest
 }) {
-  const [childrenLength, setChildrenLength] = useState<number>(0);
-  const [windowWidth, setWindowWidth] = useState<number | undefined>(undefined);
-
-  useEffect(() => {
-    setChildrenLength(items.length);
-  }, [items]);
-
-  useEffect(() => {
-    setWindowWidth(window.innerWidth);
-  }, []);
+  const [windowWidth, setWindowWidth] = useState<number | undefined>(isSsr() ? undefined : window.innerWidth);
 
   useEffect(() => {
+    if (isSsr()) return;
```

Comment:
Not directly related to this PR, but shouldn't this `useEffect` have an empty array as second argument, to only run it on init? 😄 

In its current state, I would think it runs on each re-render.